app/views/static/index.html.erb

```diff
@@ -10,7 +10,7 @@
         </h2>
 
         <% if(room = Room.recommended.take) %>
-          <%= link_to(room_path(room), class: "button is-primary is-inverted") do %>
+          <%= link_to("/rooms/random", class: "button is-primary is-inverted") do %>
```

Comment:
With the separate action in the room controller from the other comment, this can just be `random_room_path`